browser/js/common/directives/navbar/navbar.html

```diff
@@ -1,30 +1,78 @@
-<nav class="navbar navbar-static-top">
-    <div class="container">
-        <ul>
-            <li>
-                <a ng-click="goToSection('.problem')"> The Problem </a>
-            </li>
+<nav id='navbar' class='navbar navbar-static-top'>
+	<div class='container'>
+		<div class='dropdown'>
+			<div class='dropdown-title'>
+				What is Kinetic
+			</div>
+				<div class='dropdown-content'>
+					<div class='list-container'>
+						<a ng-click="goToSection('.problem', 'home')"> The Problem </a>
+						<a ng-click="goToSection('.solution', 'home')"> Our Solution </a>
+						<a ng-click="goToSection('.what-is', 'home')"> What We Do </a>
+						<a ng-click="goToSection('.about-us', 'home')"> About Us </a>
+						<a ng-click="goToSection('.sponsored', 'home')"> Key Issues </a>
+						<a ng-click="goToSection('.get-involved', 'home')"> Get Involved </a>
+					</div>
+				</div>
+		</div>
```

Comment:
I'm a little confused here by what you're saying. So we would create a data attribute for each `a` tag with the class name of where I would want the window to scroll to, but what do you mean put a data attribute on the containing `div`? I'm happy to refactor it like you said but just not exactly understanding how this would work. How would we get the data from the containing `div` in the `a` element.